agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/Indexer.java

```diff
@@ -52,7 +52,7 @@ public abstract class Indexer extends Thread {
 	protected DecimalFormat df = new DecimalFormat("#");
 	protected ObjectMapper om = new ObjectMapper();
 
-	private ProcessDisplayHelper display = new ProcessDisplayHelper();
+	protected ProcessDisplayHelper display = new ProcessDisplayHelper();
```

Comment:
Does this need to be changed? I don't see it being used in this code?

Reply:
Reverted